api/log/get_service.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+//nolint:dupl // ignore similar code with step
+package log
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-vela/server/database"
+	"github.com/go-vela/server/router/middleware/build"
+	"github.com/go-vela/server/router/middleware/org"
+	"github.com/go-vela/server/router/middleware/repo"
+	"github.com/go-vela/server/router/middleware/service"
+	"github.com/go-vela/server/router/middleware/user"
+	"github.com/go-vela/server/util"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+// swagger:operation GET /api/v1/repos/{org}/{repo}/builds/{build}/services/{service}/logs services GetServiceLog
+//
+// Retrieve the logs for a service
+//
+// ---
+// produces:
+// - application/json
+// parameters:
+// - in: path
+//   name: org
+//   description: Name of the org
+//   required: true
+//   type: string
+// - in: path
+//   name: repo
+//   description: Name of the repo
+//   required: true
+//   type: string
+// - in: path
+//   name: build
+//   description: Build number
+//   required: true
+//   type: integer
+// - in: path
+//   name: service
+//   description: ID of the service
```

Comment:
This is the service number (the number in the build) not the ID (the globally unique service ID) right?

Could we say `Number of the service in the build`? Or more succinctly `Service number` (similar to how we have `Step number` on the step log handlers).

If so, then please also do the same as `GetServiceLog` for `UpdateServiceLog` and `DeleteServiceLog`.